frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -1012,6 +986,17 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
                 actions.syncSnapshotsWithPlayer()
             }
         },
+        timestampChangeTracking: (next) => {
```

Comment:
Do we need this? I didn't see it referenced anywhere other than in the code we deleted

Reply:
so, it used to be that this was checked on every animation frame

now it will be checked each time the timestampChangeTracking changes

this might actually be the same frequency but it at least reduces the cognitive overhead of understanding `updateAnimation`